src/core/InterleavedBufferAttribute.js

```diff
@@ -91,91 +82,109 @@ class InterleavedBufferAttribute {
 
 	setX( index, x ) {
 
-		this.data.array[ index * this.data.stride + this.offset ] = x;
+		// Note: Assuming stride is multiple of type.BYTES_PER_ELEMENT
```

Comment:
Let's require (i.e. not attempt to support anything else) the requirements of glTF here, they are there for a reason:

https://github.com/KhronosGroup/glTF/blob/master/specification/2.0/README.md#data-alignment

Reply:
Oh, good to know that. I have overlooked that part in the spec. Yeah let's require unless other spec needs.